game-core/src/test/java/games/strategy/triplea/delegate/battle/steps/fire/aa/OffensiveAaFireTest.java

```diff
@@ -4,6 +4,8 @@
 import static games.strategy.triplea.delegate.battle.steps.BattleStepsTest.givenUnitWithTypeAa;
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.hasSize;
+import static org.hamcrest.collection.IsEmptyCollection.*;
+import static org.hamcrest.core.Is.*;
```

Comment:
Codacy found an issue: [Avoid unused imports such as 'org.hamcrest.core.Is'](https://app.codacy.com/gh/triplea-game/triplea/pullRequest?prid=6161498)